develop-docs/sdk/telemetry/traces/modules/queries.mdx

```diff
@@ -29,11 +29,21 @@ SELECT * FROM users WHERE id = ?;
 
 The `?` parameter is a placeholder in the query. The SDK must scrub out all values and replace them with a placeholder. Supported placeholder values are `%s`, `?`, `:c0` (e.g., `:c0`, `:c1`) and `$1` (e.g., `$1`, `$2`).
 
+For MongoDB, the description should contain valid JSON. If replacing values with a placeholder, the string `"?"` should be used.
+
+```json
+{ "find": "documents", "filter": { "wordCount": { "$gte": "?" } } }
+```
+
 ### Span Data
 
 For full support, each database span should have a `data` attribute. `data` is itself a key-value lookup of attributes. Refer to [Database Span Data Conventions](/sdk/performance/span-data-conventions/#database) for a full list of attributes that database spans should have. We recommend that the SDK provide _all_ of those attributes for every span. However, the _minimal_ requirement is that the SDK provides the `db.system` attribute.
 
-## Instrumentation Example
+#### MongoDB Data
+
+MongoDB support requires a few extra data attributes. In addition to a `db.system` value of `"mongodb"`, the `db.operation` and `db.collection.name` attributes are also required to contain the command and collection name respectively.
```

Comment:
I think we should mention that OTel and the Sentry SDKs collect these tags automatically so there's no confusion, since this line feels sort of like a CTA to me